docs/extend/modular-resources/create.md

```diff
@@ -0,0 +1,528 @@
+        if right_name == "":
+            raise Exception("A motorR attribute is required for a MyBase component.")
+        return [left_name, right_name]
+
+    # Handles attribute reconfiguration
+    def reconfigure(self, config: ComponentConfig, dependencies: Mapping[ResourceName, ResourceBase]):
+        left_name = config.attributes.fields["motorL"].string_value
+        right_name = config.attributes.fields["motorR"].string_value
+
+        left_motor = dependencies[Motor.get_resource_name(left_name)]
+        right_motor = dependencies[Motor.get_resource_name(right_name)]
+
+        self.left = cast(Motor, left_motor)
+        self.right = cast(Motor, right_motor)
+
+    """ Implement the methods the Viam RDK defines for the base API (rdk:component:base) """
+
+    # move_straight: unimplemented
+    async def move_straight(self, distance: int, velocity: float, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None, **kwargs):
+        pass
+
+    # spin: unimplemented
+    async def spin(self, angle: float, velocity: float, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None, **kwargs):
+        pass
+
+    # set_velocity: unimplemented
+    async def set_velocity( self, linear: Vector3, angular: Vector3, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None, **kwargs):
+        pass
+
+    # set_power: set the linear and angular velocity of the left and right motors on the base
+    async def set_power(self, linear: Vector3, angular: Vector3, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None, **kwargs):
+
+        # stop the base if absolute value of linear and angular velocity is less than .01
+        if abs(linear.y) < 0.01 and abs(angular.z) < 0.01:
+            return self.stop(extra=extra, timeout=timeout)
+
+        # use linear and angular velocity to calculate percentage of max power to pass to SetPower for left & right motors
+        sum = abs(linear.y) + abs(angular.z)
+
+        self.left.set_power(power=((linear.y - angular.z) / sum), extra=extra, timeout=timeout)
+        self.right.set_power(power=((linear.y + angular.z) / sum), extra=extra, timeout=timeout)
+
+    # stop: stop the base from moving by stopping both motors
+    async def stop(self, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None, **kwargs):
+        self.left.stop(extra=extra, timeout=timeout)
+        self.right.stop(extra=extra, timeout=timeout)
+
+    # is_moving: check if either motor on the base is moving with motors' is_powered
+    async def is_moving(self, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None, **kwargs) -> bool:
+        return self.left.is_powered(extra=extra, timeout=timeout)[0] or self.right.is_powered(extra=extra, timeout=timeout)[0]
+```
+
+<br>
+</details>
+
+<file>__init__.py</file> registers the mybase custom model and API helper functions with the SDK.
```

Comment:
right now in the rendering, init is showing up bold instead of with the underscores.